packages/google_sign_in/google_sign_in_web/example/integration_test/flexible_size_html_element_view_test.dart

```diff
@@ -200,10 +200,8 @@ void resize(web.HTMLElement resizable, Size size) {
 }
 
 /// Returns a function that can be used to inject `element` in `onPlatformViewCreated` callbacks.
```

Comment:
`onElementCreated`

Reply:
Good eye, I'll do a quick find-and-replace just in case some more slipped through the cracks.